.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,430 @@
+name:        "Bug Report"
+description: "Create a new Bug report. Have you found an unexpected behavior?, use this form."
```

```suggestion
description: "Create a new Bug report. Have you found an unexpected behavior? Use this form."
```
`?` and `!` mark the end of a sentence, so it's incorrect to continue with a `,` afterwards.